### PR TITLE
This is a finished mobile version project

### PR DESCRIPTION
**Pull request by rcniyigena**
This is a finished portfolio project for mobile version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portifolio</title>
+  <link rel="stylesheet" href="style.css" />
+  <!-- <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'> -->
```
Comment (wayungi) [inline index.html]: - [ ] Kindly remove all commentes code. They make the repo dirty

File: style.css [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+
+  .tonic_div {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+
+    /* width: 295px;
+    height: 80px; */
```
Comment (wayungi) [inline style.css]: - [ ] More commnets here. Please delete all commented code to make your repo clean 😄

File: index.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
```
Comment (wayungi) [inline index.html]: - [ ] Please make sure the button color matches the figma design. 
**figma**
btn1

**your design**
btn2

File: index.html [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+        </div>
+      </div>
+    </section>
```
Comment (wayungi) [inline index.html]: - [ ] All your projects should be grouped in a section tag and styled according to requirements.
inst

File: style.css [hunk 5]
```diff
@@ -0,0 +1,385 @@
...
+    padding: 16px;
+    gap: 12px;
+    width: 327px;
+    height: 552px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    margin-top: 130px;
+    justify-content: space-between;
+  }
```
Comment (wayungi) [inline style.css]: - [ ] This is the project card. it should be aligned with Flexbox not grid. 😄 
inst

File: index.html [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
```
Comment (wayungi) [inline index.html]: - [ ] Please make sure your `see project button` has the sudo state of pressed implemented.
states

File: index.html [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+      </h2>
+        <p>Hello I'm software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <div class="about_social_media1">
+        <h2 class="connect">LET'S CONNECT</h2>
+        <div class="about_social">
+          <a href=""><img src="images/Vector5.png" alt=""></a>
+          <a><img src="images/Linkedin icon.png" alt=""></a>
+          <a><img src="images/Vector4.png" alt=""></a>
+          <a><img src="images/Vector2.png" alt=""></a>
+          <a><img src="images/Vector3.jpg" alt=""></a>
```
Comment (wayungi) [inline index.html]: - [ ] Please use `ul` & `li` tags to group the icons. This is a project requirement. 👍🏾

File: index.html [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+        </div>
+        <div class="view_button">
+          <button>Get my resume</button>
```
Comment (wayungi) [inline index.html]: - [ ] Kindly make sure this button has the **pressed state** style.
states

File: index.html [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+        <div class="language">
+          <h2>Frameworks</h2>
+          <img src="images/Union_right.png" alt="drop down image">
+          </div>
+          <img src="images/Separator Bottom.png" alt="horizontal line">
+          <div class="language">
+            <h2>Skills</h2>
+            <img src="images/Union_right.png" alt="drop down image">
+            </div>
+            <img class="hr_line" src="images/Separator Bottom.png" alt="horizontal line">
```
Comment (wayungi) [inline index.html]: - [ ] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li, and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li. This is a project requirement that ensures your code is well structured. 🤝

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @rcniyigena , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 10]
```diff
@@ -0,0 +1,395 @@
...
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    color: #396df2;
+    text-align: center;
+    letter-spacing: 0.03em;
+    font-weight: bold;
+    background: none;
+  }
+
```
Comment (Johnadibe) [inline style.css:205]: > - [ ] Great work @rcniyigena👍 As requested by the previous code reviewer, I noticed that you did not implement the hover and pressed state for the See project and Get my Resume buttons interaction. This is to provide visual feedback to users when they interact with the button. When a user hovers over a button, it should change in some way to indicate that it is clickable. Similarly, when a user presses down on the button, it should change again to provide feedback that the button has been activated. This feedback helps users understand how to interact with the button and provides a better user experience. Without these visual cues, users may be unsure whether a button is clickable or not, which can lead to frustration and confusion. So please kindly do that😊
> 
> Please check the Figma design:
> 
> fig
>

File: index.html [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+            <li><h2>Frameworks</h2></li>
+            <li><img src="images/Union_right.png" alt="drop down image"></li>
+          </ul>
+            <img src="images/Separator Bottom.png" alt="horizontal line">
+            <ul class="language">
+              <li><h2>Skills</h2></li>
+              <li><img src="images/Union_right.png" alt="drop down image"></li>
+            </ul>
+              <img class="" src="images/Separator Bottom.png" alt="horizontal line">
+        </div>
```
Comment (Johnadibe) [inline index.html]: > - [ ] Great job on implementing some changes👍. However, as requested by the previous reviewer, I noticed that the technology categories ("Languages", "Frameworks", and "Skills") are not wrapped in a ul with each of them in a li. You have already wrapped the technologies in each category (e.g., "JavaScript", "HTML", and "CSS") in a ul with each of them in a li👍. Please kindly make this adjustment to ensure consistency with the requested design.
> 
> **You can use the following example for reference:**
> ```
> <ul>
>   <li>
>     Languages
>     <ul>
>       <li>JavaScript</li>
>       <li>HTML</li>
>       <li>CSS</li>
>     </ul>
>   </li>
>   <li>
>     Frameworks
>     <ul>
>       <li>Ruby on Rails</li>
>       <li>React</li>
>       <li>Angular</li>
>     </ul>
>   </li>
>   <li>
>     Skills
>     <ul>
>       <li>Problem-solving</li>
>       <li>Collaboration</li>
>       <li>Time management</li>
>     </ul>
>   </li>
> </ul>
> 
> ```

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
### Hi @rcniyigena🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Good commit history ✔️
- No linter errors✔️
- Your project is documented professionally✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rcniyigena** [comment]
Hello Team,

I made a changes please take a look.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @rcniyigena , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**